README.md

```diff
@@ -186,7 +187,7 @@ deployment:
   itunes_connect_prod:
     branch: release/beta
     commands:
-      - bin/execute.sh itunes-connect --scheme MyProject-PROD
+     - bin/execute.sh itunes-connect --scheme MyProject-PROD
```

Comment:
oops. fyi this was a branch i was using to do testing with circle (and buildkite) over the xmas/nye period when we had a bit of time to invest in things like this. i've also been in touch with circle re: billing, and had a call from their customer service people (sounded serious!) on friday while i was on holidays. I plan to speak to them some time this week to see if they can give us an update on why the simulator/tests are failing intermittenlty